### PR TITLE
XWIKI-13843: Drop support for IE6

### PR DESCRIPTION
**Pull request by alexhenrie**
Virtually all of this was dead code, since on current browsers window.browser.isIE6x is false and the CSS "* html" does not match any elements. The entire csshover3-source.htc file was an IE6 hack, but it was not actually used anywhere. I did not delete anything in third-party components that XWiki relies on and I left all the IE7 hacks alone.

### CODE REVIEW

**Comment by mflorea** [comment]
Thanks!

File: xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/js/xwiki/table/livetable.css [hunk 1]
```diff
@@ -179,13 +169,6 @@ thead.xwiki-livetable-display-header td.xwiki-livetable-display-header-filter se
 tbody.xwiki-livetable-display-body tr {
   border-bottom: 1px solid $theme.borderColor;
 }
-tbody.xwiki-livetable-display-body tr:hover , tbody.xwiki-livetable-display-body tr.rowHover { /* for IE6 tr:hover fix*/
```
Comment (sdumitriu) [inline xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/js/xwiki/table/livetable.css]: This had both general and ie6 selectors, the rule should be put back, but only with the first selector.

Comment (mflorea) [inline xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/js/xwiki/table/livetable.css]: Indeed. I fixed it. Thanks @sdumitriu!

Comment (alexhenrie) [inline xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/js/xwiki/table/livetable.css]: Thanks for putting back "thead.xwiki-livetable-display-header tr.xwiki-livetable-display-filters:hover" too, and sorry for missing these. Without all the IE6 hacks, the code is much more understandable now. Thank you!

Comment (alexhenrie) [inline xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/js/xwiki/table/livetable.css]: We should also put back "thead.xwiki-livetable-display-header tr.xwiki-livetable-display-filters:hover" too, sorry for missing these. I've pushed a fix to my branch. (I thought you had fixed both, then realized that you had only fixed the one.)

Comment (mflorea) [inline xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/js/xwiki/table/livetable.css]: Done. Thanks!

**Pull request merged**